### ISSUE TITLE
Bracelet LEDs should be easier to assemble

### ISSUE THREAD

**Issue by griswaldbrooks** [kestrel-labs/superspreader#19]
### Need

_In order to_ make boards easier to assemble
_As an_ board assembler
_I want_ the board to have fewer THT components

### Acceptance Criteria

**Scenario:** LEDs should be surface mount
**GIVEN** bracelet components
**WHEN** assembling
**THEN** LEDs should be an SMD component

### Tasks

- [ ] Find low current surface mount LEDs
- [ ] Replace current bi-color LED with two LEDs #22 
- [ ] Route

### Notes
Basically, all of the LEDs should be surface mount

References:
